sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -401,4 +404,14 @@ private boolean doesArgsHaveThrowable(Object... args) {
     private Object[] removeThrowable(Object... args) {
         return Arrays.copyOf(args, args.length - 1);
     }
+
+    /**
+     * Removes CRLF pattern in the {@code logMessage}.
+     *
+     * @param logMessage The log message to sanitize.
+     * @return The updated logMessage.
+     */
+    private static String sanitizeLogMessageInput(String logMessage) {
+        return CRLF_PATTERN.matcher(logMessage).replaceAll("");
```

Comment:
So, effectively we aren't allowing Windows style newlines in logging?

Reply:
it is either of \r or \n.
The solution is recommended here: https://find-sec-bugs.github.io/bugs.htm#CRLF_INJECTION_LOGS